src/components/WorkerList/index.tsx

```diff
@@ -1,20 +1,32 @@
 'use client';
 
+import { useState } from 'react';
+
 import * as S from './style';
 
-import { WorkerListItem } from '@/components';
+import { SearchBar, WorkerListItem } from '@/components';
 import type { WorkerType } from '@/types';
 
 interface Props {
-  workerList: WorkerType[];
+  initWorkerList: WorkerType[];
 }
 
-const WorkerList: React.FC<Props> = ({ workerList }) => (
-  <S.WorkerList>
-    {workerList.map((worker) => (
-      <WorkerListItem key={worker.id} worker={worker} />
-    ))}
-  </S.WorkerList>
-);
+const WorkerList: React.FC<Props> = ({ initWorkerList }) => {
+  const [workerList, setWorkerList] = useState(initWorkerList);
```

Comment:
```suggestion
  const [workerList, setWorkerList] = useState<WorkerType[]>(initWorkerList);
```

타입 지정 가능해보입니다 확인 부탁해요

Reply:
반영하겠습니다!